### PR TITLE
[fix] Not converted type

### PR DESCRIPTION
**Pull request by heckad**
https://github.com/mosquito/aio-pika/issues/260

### CODE REVIEW

File: aio_pika/channel.py [hunk 1]
```diff
@@ -210,6 +210,8 @@ def _on_return(self, message: aiormq.types.DeliveredMessage):
 
         if auto_delete and durable is None:
             durable = False
+        
+        type = ExchangeType(type)
```
Comment (mosquito) [inline aio_pika/channel.py]: This was done intentionally. The fact is that in RabbitMQ, extensions use custom Exchange types other than this enumeration. So this check breaks the use of third-party Exchange types.

Comment (heckad) [reply aio_pika/channel.py]: Then you have the wrong type hint. I tried to send a string and got an error. Look [#260](https://github.com/mosquito/aio-pika/issues/260).

**Comment by heckad** [comment]
Why did tests fail in different places? My fix can't break it.

File: aio_pika/channel.py (hunk 1 above)
Comment (heckad) [reply aio_pika/channel.py]: @mosquito ?

**Comment by mosquito** [comment]
Please rebase against master

**Pull request closed without merging**